lldb/test/API/tools/lldb-server/TestLldbGdbServer.py

```diff
@@ -5,9 +5,9 @@
 lldb-server tests run where the lldb-server exe is
 available.
 
-This class will be broken into smaller test case classes by
-gdb remote packet functional areas.  For now it contains
-the initial set of tests implemented.
+The tests are split between the LldbGdbServerTestCase and
+LldbGdbServerTestCase2 classes to avoid timeouts in the
+test suite.
```

Comment:
Maybe expand this so folks don't get the wrong idea in future. Perhaps:
"lit's timeout applies to the whole file, rather than each test_ method, so these tests are split between...."

Reply:
Good idea. Updated the comment.